test/phoenix_digest_test.exs

```diff
@@ -15,6 +16,9 @@ defmodule BootlegPhoenix.PhoenixDigestTest do
     build_host = List.first(hosts)
     app_hosts = hosts -- [build_host]
 
+    Logger.info "build host is #{container_id(build_host)}"
```

Comment:
This sort of code doesn't belong in a specific test case. If you wanted to have `TEST_VERBOSE` dump the entire bootleg role config to `debug`, that'd be 👍 .